src/session/content/message_row/document.rs

```diff
@@ -243,4 +251,37 @@ impl MessageDocument {
             }
         }
     }
+
+    fn try_load_thumbnail(&self, message: &Message) {
+        if let MessageContent::MessageDocument(data) = message.content().0 {
+            let imp = self.imp();
+            if let Some(thumbnail) = data.document.thumbnail {
+                imp.file_thumbnail_picture.set_visible(true);
+                imp.file_box.add_css_class("with-thumbnail");
```

Comment:
Why not adding the class to `MessageDocument` itself?

Reply:
I want to have the same style class for grouped documents, so it should be used for each file separately